apps/prairielearn/src/pages/instructorAssessmentQuestionStatistics/instructorAssessmentQuestionStatistics.html.ts

```diff
@@ -94,7 +94,9 @@ export function InstructorAssessmentQuestionStatistics({
 
           <div class="card mb-4">
             <div class="card-header bg-primary text-white d-flex align-items-center">
-              ${resLocals.assessment_set.name} ${resLocals.assessment.number}: Question difficulty
+              <h2>
+                ${resLocals.assessment_set.name} ${resLocals.assessment.number}: Question difficulty
+              </h2>
```

Comment:
This could possibly be improved when we tackle https://github.com/PrairieLearn/PrairieLearn/issues/10447. I don't think it's necessary to repeat the name/number for every card on the page. Not requesting a change here; let's discuss separately.

Reply:
That is a great point. Let me know when you want to discuss these headings.

Reply:
Maybe add this as a note to that issue? I think we can discuss some of this in a future dev meeting.

Reply:
Added a note to that issue.